app/overrides/hosts_list.rb

```diff
@@ -2,7 +2,7 @@
   virtual_path: 'hosts/_list',
   name: 'insights_hits_header',
   insert_before: 'thead tr th.hidden-xs:first-of-type',
-  text: '<th class="hidden-xs" width="10%"><%= _("Insights")%></th>'
+  text: '<th class="hidden-xs" width="10%"><%= sort :"insights_recommendations_count", :as => _("Insights")%></th>'
```

Comment:
```suggestion
  text: '<th class="hidden-xs" width="10%"><%= sort :insights_recommendations_count, :as => _("Insights")%></th>'
```

No need for string here.